py-polars/polars/dataframe/frame.py

```diff
@@ -5753,9 +5753,11 @@ def drop_in_place(self, name: str) -> Series:
 
     def clear(self, n: int = 0) -> Self:
         """
-        Create an empty copy of the current DataFrame, with zero to 'n' rows.
+        Create an empty copy of the current DataFrame, with `n` rows.
```

Comment:
Taking this out doesn't seem to add anything; zero is what you get if you don't explicitly set `n`, and it also defines the -admittedly obvious- a lower bound (as you can't have negative rows).

Reply:
The precise way it is written currently doesn't tell you how many rows precisely will be in the dataframe. The dataframe will contain precisely n rows. 

Saying it will contain "0 to n rows" means it may contain 0, or 1, or 2, or n-2, or n-1, or n rows. 

The default argument is captured in the function definition of course, but a better way to describe the default would be to write something like, "By default, ´n´=0"

Reply:
How about:

> Create an empty copy of the current DataFrame. If `n` > 0, create `n` null-filled rows.

There's a small issue with "null" and "empty." I think of an empty frame as one without rows, but this method can also create a non-empty frame, just one filled with nulls.